scene/resources/material.h

```diff
@@ -439,6 +439,8 @@ class SpatialMaterial : public Material {
 
 	Ref<Texture> textures[TEXTURE_MAX];
 
+	bool force_vertex_shading;
```

Comment:
Initialize to false to be safe.

Reply:
Can we do that in 3.2?

Reply:
Sure, it uses C++14 (we didn't officially modernized the codebase, but we can still use relevant modern features like this).

Reply:
Okay!